File: Algorithms/Easy/561_ArrayPartitionI/Solution.cs
public class Solution {
    public int ArrayPairSum(int[] nums) {
        int n = nums.Length, answer = 0;
        Array.Sort(nums);
        
        for (int i = 0; i < n; i += 2)
            answer += nums[i];
        
        return answer;
    }
}

File: Algorithms/Easy/561_ArrayPartitionI/Solution.java
class Solution {
    public int arrayPairSum(int[] nums) {
        int n = nums.length, answer = 0;
        Arrays.sort(nums);
        
        for (int i = 0; i < n; i += 2)
            answer += nums[i];
        
        return answer;
    }
}